### PR TITLE
Added 'fr' and 'es' i18n messages.

### PR DESCRIPTION
**Pull request by PremnishaTW**
Added french and spanish translation values

**Sample screenshot for french**
image

**Sample screenshot for spanish**
image

@mks-d Please review the changes

### CODE REVIEW

**Review by mks-d — APPROVED** [review]
@brandones any chance you can check the 'es' translations?

**Comment by brandones** [comment]
Sure. @PremnishaTW , can you post screenshots that capture the changes you made? The screenshots you posted are of the patient dashboard (which has one of those translation strings, but there are many others). I'm not familiar enough with the attachments module to know what all of these messages are.

**Comment by PremnishaTW** [comment]
@brandones Please find the attachments for reference

French:
image

Spanish:
image

File: api/src/main/resources/messages_es.properties [hunk 1]
```diff
@@ -1 +1,32 @@
+${project.parent.artifactId}.title=Módulo adjuntos
+${project.parent.artifactId}.manage=Manejo de módulo
 
+${project.parent.artifactId}.visitactions.label=Archivos adjuntos
+
+${project.parent.artifactId}.breadcrumbs.label=Archivos adjuntos
+
+${project.parent.artifactId}.dropzone.innerlabel=Haga clic o soltar un archivo aquí.
```
Comment (brandones) [inline api/src/main/resources/messages_es.properties]:
```suggestion
${project.parent.artifactId}.dropzone.innerlabel=Haga clic o deje un archivo aquí.
```

File: api/src/main/resources/messages_es.properties [hunk 2]
```diff
@@ -1 +1,32 @@
...
+
+${project.parent.artifactId}.attachmentspage.fileTitle=Expediente
+${project.parent.artifactId}.attachmentspage.commentTitle=Subtítulo
```
Comment (brandones) [inline api/src/main/resources/messages_es.properties]:
```suggestion
${project.parent.artifactId}.attachmentspage.commentTitle=Comentario
```

File: api/src/main/resources/messages_es.properties [hunk 3]
```diff
@@ -1 +1,32 @@
...
+${project.parent.artifactId}.attachmentspage.galleryTitle=
+${project.parent.artifactId}.attachmentspage.uploadButton=Subir archivo
+${project.parent.artifactId}.attachmentspage.clearFormsButton=formas claras
```
Comment (brandones) [inline api/src/main/resources/messages_es.properties]:
```suggestion
${project.parent.artifactId}.attachmentspage.clearFormsButton=Borrar todo
```

File: api/src/main/resources/messages_es.properties [hunk 4]
```diff
@@ -1 +1,32 @@
...
+${project.parent.artifactId}.attachmentspage.delete.title=Eliminar archivo adjunto
```
Comment (brandones) [inline api/src/main/resources/messages_es.properties:15]:
```suggestion
${project.parent.artifactId}.attachmentspage.delete.title=Eliminar archivo adjunto
```

File: api/src/main/resources/messages_es.properties [hunk 5]
```diff
@@ -1 +1,32 @@
...
+${project.parent.artifactId}.attachmentspage.delete.confirm=¿Seguro que desea eliminar este apego?
```
Comment (brandones) [inline api/src/main/resources/messages_es.properties]:
```suggestion
${project.parent.artifactId}.attachmentspage.delete.confirm=¿Seguro que desea eliminar este archivo adjunto?
```

File: api/src/main/resources/messages_es.properties [hunk 6]
```diff
@@ -1 +1,32 @@
+${project.parent.artifactId}.title=Módulo adjuntos
+${project.parent.artifactId}.manage=Manejo de módulo
 
+${project.parent.artifactId}.visitactions.label=Archivos adjuntos
+
+${project.parent.artifactId}.breadcrumbs.label=Archivos adjuntos
+
+${project.parent.artifactId}.dropzone.innerlabel=Haga clic o soltar un archivo aquí.
+
+${project.parent.artifactId}.attachmentspage.fileTitle=Expediente
```
Comment (brandones) [inline api/src/main/resources/messages_es.properties]:
```suggestion
${project.parent.artifactId}.attachmentspage.fileTitle=Archivo
```

File: api/src/main/resources/messages_es.properties [hunk 7]
```diff
@@ -1 +1,32 @@
...
+${project.parent.artifactId}.attachmentspage.commentTitle=Subtítulo
+${project.parent.artifactId}.attachmentspage.galleryTitle=
+${project.parent.artifactId}.attachmentspage.uploadButton=Subir archivo
+${project.parent.artifactId}.attachmentspage.clearFormsButton=formas claras
+${project.parent.artifactId}.attachmentspage.delete.title=Eliminar archivo adjunto
+${project.parent.artifactId}.attachmentspage.delete.confirm=¿Seguro que desea eliminar este apego?
+
+${project.parent.artifactId}.fileUpload.success=El archivo adjunto se ha subido correctamente.
+${project.parent.artifactId}.fileUpload.error=Se produjo un error y el archivo adjunto no se pudo cargar.
+${project.parent.artifactId}.fileUpload.attentionPastVisit=¡Atención! Los accesorios se pueden cargar en una visita pasado.
```
Comment (brandones) [inline api/src/main/resources/messages_es.properties]:
```suggestion
${project.parent.artifactId}.fileUpload.attentionPastVisit=¡Atención! Los accesorios estan cargando en una visita pasada.
```

File: api/src/main/resources/messages_es.properties [hunk 8]
```diff
@@ -1 +1,32 @@
...
+
+${project.parent.artifactId}.gallery.loadMore=Carga más
+
+${project.parent.artifactId}.thumbail.get.error=Se ha producido un error al obtener el archivo adjunto.
+${project.parent.artifactId}.thumbail.save.success=El archivo adjunto se ha actualizado correctamente.
+${project.parent.artifactId}.thumbail.save.error=Se produjo un error y el archivo adjunto no pudo ser actualizado.
+${project.parent.artifactId}.thumbail.delete.success=La fijación se ha eliminado correctamente.
```
Comment (brandones) [inline api/src/main/resources/messages_es.properties]:
```suggestion
${project.parent.artifactId}.thumbail.delete.success=El archivo se ha eliminado correctamente.
```

File: api/src/main/resources/messages_es.properties [hunk 9]
```diff
@@ -1 +1,32 @@
...
+${project.parent.artifactId}.thumbail.delete.error=Se ha producido un error y la unión no se pudo eliminar.
```
Comment (brandones) [inline api/src/main/resources/messages_es.properties]:
```suggestion
${project.parent.artifactId}.thumbail.delete.error=Se ha producido un error y el archivo adjunto no se pudo eliminar.
```

File: api/src/main/resources/messages_es.properties [hunk 10]
```diff
@@ -1 +1,32 @@
...
+
+${project.parent.artifactId}.misc.label.enterCaption=Introducir un título
```
Comment (brandones) [inline api/src/main/resources/messages_es.properties]:
```suggestion
${project.parent.artifactId}.misc.label.enterCaption=Introducir un comentario
```

**Review by brandones — CHANGES_REQUESTED** [review]
I couldn't find this in Transifex so I added the correct translations here.

**Comment by brandones** [comment]
@PremnishaTW  @mks-d Will these translations propagate to Transifex?

**Pull request merged**